smac/intensification/intensification.py

```diff
@@ -112,13 +110,34 @@ def __init__(self, tae_runner: ExecuteTARun,
         self.use_ta_time_bound = use_ta_time_bound
         self.min_chall = min_chall
 
-    def intensify(self, challengers: typing.Optional[typing.List[Configuration]],
-                  optimizer: typing.Optional['SMBO'],
-                  incumbent: Configuration,
-                  run_history: RunHistory,
-                  aggregate_func: typing.Callable,
-                  time_bound: float = float(MAXINT),
-                  log_traj: bool = True) -> typing.Tuple[Configuration, float]:
+        # stage variables
+        # the intensification procedure is divided into 3 'stages':
+        # 1. add incumbent run
```

Comment:
could you please add line numbers for the Intensify pseudo-code from the SMAC paper to the different variables??